gammapy/scripts/tests/test_image_fit.py

```diff
@@ -44,5 +48,97 @@ def test_sherpa_like(tmpdir):
     # old: [  9.016334  ,  99.365574   ,  99.647234   , 10.97365    ]
     # new: [ 10.7427035 ,  98.16618776 ,  98.45487028 ,  7.73529899 ]
     actual = data['fit']['parvals']
-    expected = [10.7427035, 98.16618776, 98.45487028, 7.73529899]
+    expected = [9.016526, 99.865985, 100.147877, 1010.824189]
     assert_allclose(actual, expected)
+
+
+@requires_dependency('sherpa')
+@requires_data('gammapy-extra')
+def test_sherpa_like_no_psf_model(tmpdir):
+    """
+    Fit model image without PSF.
+    """
+    # load test data
+    filenames = load_poisson_stats_image(extra_info=True, return_filenames=True)
+    outfile = tmpdir / 'test_sherpa_like.json'
+
+    # write test source json file
+    sources_data = {}
+    sources_data['gaussian'] = {'ampl': 1E3,
+                                'xpos': 99,
+                                'ypos': 99,
+                                'fwhm': 4 * 2.3548}
```

Comment:
You're using 2.3548 as a magic constant in a few places in this PR.
Please use this constant instead:
```python
>>> from astropy.stats import gaussian_fwhm_to_sigma, gaussian_sigma_to_fwhm
>>> gaussian_fwhm_to_sigma, gaussian_sigma_to_fwhm
(0.42466090014400953, 2.3548200450309493)
```
Would you mind removing the version we have in Gammapy with the one from Astropy in this PR?
https://github.com/gammapy/gammapy/blob/0cfea1a3e402ebf1effb22045aabbc91b99ea1e8/gammapy/utils/const.py#L9
https://github.com/gammapy/gammapy/search?utf8=%E2%9C%93&q=fwhm&type=Code